Exercises/01/Begin/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 cmake_minimum_required(VERSION 3.16)
 project(KokkosTutorial01)
-include(../../common.cmake)
+
+# Add a custom module path for find_package
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/../../.cmake)
+
+find_package(Kokkos REQUIRED)
+
+if (Kokkos_DEVICES)
+    message(WARNING "${CMAKE_CURRENT_SOURCE_DIR}"
+            "Kokkos device is enabled, it might cause issue with the current program")
+endif ()
```

Comment:
This could also just be host backends like `SERIAL` and `OPENMP`. You really want to check for GPU backends, right?

Reply:
Yes, and THREADS, and perhaps more?
In fact, I want to warn for backends for which the default memory space can be unreachable from the host.